### PR TITLE
Added documentation to CounterHandler.java

### PR DESCRIPTION
**Pull request by SureSRM**
Added comments for every member and method. Possible bug detected will be treated in another issue

### CODE REVIEW

File: src/main/java/es/unizar/webeng/hello/CounterHandler.java [hunk 1]
```diff
@@ -24,19 +24,33 @@
 @Component
 public class CounterHandler extends TextWebSocketHandler {
 
+    /** System's logger for exception notifying.
```
Comment (fjlopez) [inline src/main/java/es/unizar/webeng/hello/CounterHandler.java]: Javadoc blocks should be as follows:
```java
/**
 * Text
 */
```

File: src/main/java/es/unizar/webeng/hello/CounterHandler.java [hunk 2]
```diff
@@ -24,19 +24,33 @@
...
     private ConcurrentMap<String, WebSocketSession> activeSessions;
 
+    /** ObjectMapper that converts HashMap to JSON (used in broadcastSessionCount() ).
+     */
     @Autowired
     private ObjectMapper mapper;
 
+    /**
+     * This method initializes the class after doing the autowiring because of the
+     * Java tag @PostConstruct.
```
Comment (fjlopez) [inline src/main/java/es/unizar/webeng/hello/CounterHandler.java]: Add a javadoc `@link` to the documentation of `@PostConstruct`.

**Comment by fjlopez** [comment]
You must sync first with the master branch as there are conflicts with the `README.md`file.

File: src/main/java/es/unizar/webeng/hello/CounterHandler.java (hunk 2 above)
Comment (SureSRM) [inline src/main/java/es/unizar/webeng/hello/CounterHandler.java]: I din't fully understand this comment but finally I think I get it. Let me know if there is still wrong please.

**Pull request merged**